### PR TITLE
Increase limits AAD 1-3 and create AAD-4

### PR DESCRIPTION
**Pull request by jwade1327**
screencapture-localhost-22290-templates-aad-enewsletter-1-2023-02-27-08_17_21
screencapture-localhost-22290-templates-aad-enewsletter-4-expanded-2023-02-27-08_53_29

### CODE REVIEW

**Comment by jwade1327** [comment]
[New AAD eNewsletter Template](https://trello.com/c/unb1tOAx/437-new-aad-enewsletter-template)

**Comment by jwade1327** [comment]
[AAD Newsletter Template Update](https://trello.com/c/yHys5rok/438-aad-newsletter-template-update)

File: tenants/all/templates/aad-enewsletter-3.marko [hunk 1]
```diff
@@ -219,6 +219,74 @@ $ const nativeX = config.getAsObject("nativeX");
...
+
+        <!-- Content list block -->
+        <common-content-list-block
+          date=date
+          section-name="Main"
+          newsletter=newsletter
+          skip=14
+          storyLocation=15
+        />
+
```
Comment (Shinsina) [inline tenants/all/templates/aad-enewsletter-3.marko:289]: Noticed that `tenants/all/template/aad-newsletter-4-expanded.marko` only has 13 ad slots and doesn't seem to line up with the rest of these adjustments is this intentional?

File: tenants/all/templates/aad-enewsletter-4-expanded.marko [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+          storyLocation=15
+        />
+
+        <!--Ad Slot 13 -->
+        <common-ad-emailx-block
+          newsletter=newsletter
+          ad-unit=emailX.getAdUnit({ name: 'ad-slot-13', alias: newsletter.alias })
+          date=date
+          link-label-props={ adLocation: 15, platformDesignator }
+        />
```
Comment (Shinsina) [inline tenants/all/templates/aad-enewsletter-4-expanded.marko:298]: I'm confused are these link-label-props still being used by anything and if they are, are these both supposed to be 15 or is this a mistake?

**Review by Shinsina — CHANGES_REQUESTED** [review]

File: tenants/all/templates/aad-enewsletter-4-expanded.marko (hunk 2 above)
Comment (jwade1327) [reply tenants/all/templates/aad-enewsletter-4-expanded.marko]: @Shinsina Fixed.

**Review by Shinsina — APPROVED** [review]

**Pull request merged**